devito/operator.py

```diff
@@ -163,6 +163,8 @@ def __init__(self, expressions, **kwargs):
         # Expression lowering: evaluation of derivatives, indexification,
         # substitution rules, specialization
         expressions = [i.evaluate for i in expressions]
+        # split vector equation to list of equation
```

Comment:
this comment should be squashed with the one above --
`Expression lowering: evaluation of derivatives, flattening, indexification, ...`